articles/annotation_and_query.md

```diff
@@ -13,98 +13,85 @@ category: howto
 
-In this case, the annotations you may want to add might look like this: 
+ For example, if you have uploaded a collection of alignment files in the BAM file format from an RNA-sequencing experiment, each representing a sample and experimental replicate, you can use annotations to add this information to each file in a structured way. Sometimes, users encode this information in file names, e.g., `sampleA_conditionB.bam`, which makes it "human-readable" but makes it difficult to search for in a systematic way, such as finding all replicates of `sampleA_conditionB`. Adding this information as Synapse annotations enables a more complete description of the contents of the `File`. 
 
-* `assay = RNA-seq`
-* `fileType = bam`
-* `sample = 1`
-* `condition = A`
+ In this case, the annotations you may want to add might look like this: 
 
-For the second sample in the same experimental set with the same conditions, the annotations would have the same "keys" but slightly different "values", such as:
-
-* `assay = RNA-seq`
-* `fileType = bam`
-* `sample = 2` (note that we've changed the value here but the rest are the same)
-* `condition = A`
-
- If these samples were part of a dataset with multiple assays, such as RNA-seq and ATAC-seq, you would annotate the file entities with either `assay = RNA-seq` or `assay = ATAC-seq`. 
+ <img src="/assets/images/annotationsComplete.png">
 
 ## Types of Annotations
 
-Annotations can be one of four basic types 
+Annotations can be one of four basic types: 
 
 * Text (Character Limit = 256)
 * Integer
 * Floating Point
 * Date (Date and Time stored as a Timestamp)
 
+<br/>
 You cannot have duplicate annotation keys on the same object in Synapse. 
 
-### How to Assign Annotations
+## How to Assign Annotations
 
-Annotations may be added when initially uploading a file, or at a later date. This can be done using the command line client, the Python client, the R client, or from the Web. Using the analytical clients facilitates batch and automated population of annotations across many files. The Web client is useful when uploading a single file, or if a minor change needs to be made to annotations on a few files.
+Annotations may be added when initially uploading a file, or at a later date. This can be done using the command line client, the Python client, the R client, or from the Web. Using the analytical clients facilitates batch and automated population of annotations across many files. The Web client can be used to bulk update many files using [file views](/articles/views.html).
```

Comment:
```suggestion
Annotations may be added when initially uploading a file or at a later date. This can be done using the command line client, the Python client, the R client, or from the web. Using the programmatic clients facilitates batch and automated population of annotations across many files. The web client can be used to bulk update many files using [file views](/articles/views.html).
```

Reply:
The web client can't easily bulk update (e.g. can't find and replace an entire column). It does the action in bulk, but all of the work would be manual, and the table edit interface isn't that helpful.

Reply:
So, @kdaily , are you suggesting I provide more context to the description?